test/parallel/test-buffer-alloc.js

```diff
@@ -804,20 +804,52 @@ assert.strictEqual(Buffer.from('13.37').length, 5);
 Buffer.from(Buffer.allocUnsafe(0), 0, 0);
 
 // issue GH-5587
-assert.throws(() => Buffer.alloc(8).writeFloatLE(0, 5), RangeError);
-assert.throws(() => Buffer.alloc(16).writeDoubleLE(0, 9), RangeError);
+common.expectsError(
+  () => Buffer.alloc(8).writeFloatLE(0, 5),
+  {
```

Comment:
The following object can be stored in a variable instead (say `outOfRangeError`):
```
{
    code: 'ERR_OUT_OF_RANGE',
    type: RangeError
 }
```

This way `outOfRangeError` can be passed as follows:
`common.expectsError(<Function>, outOfRangeError)`

Reply:
Makes sense. Pushing up a new commit soon.